syntax = "proto3";

package transport_router_serialize;

message RoutingSettings {
    int32 bus_wait_time = 1;
    int32 bus_velocity = 2;
}